native/opencv-gpu.cc

```diff
@@ -26,7 +25,7 @@ void* cv_mat_from_gpu_mat(cv::cuda::GpuMat* gpu_image) {
 }
 
 void* cv_gpu_mat_from_mat(cv::Mat* image) {
-  return new cv::cuda::GpuMat(*image);
+    return new cv::cuda::GpuMat(*image);
```

Comment:
especially when it actually mix single and multiline styles for unknown reason.

Reply:
I think this line would exceed 80 if put on a single line. But you are right, the good style should always have multiple lines for this situation.